references/video_classification/train.py

```diff
@@ -389,3 +375,4 @@ def parse_args():
 if __name__ == "__main__":
     args = parse_args()
     main(args)
+
```

Comment:
Remove this extra line because the [linter](https://app.circleci.com/pipelines/github/pytorch/vision/13063/workflows/2f581b61-a1db-4d2a-b182-a90538cf38b8/jobs/1041825) is complaining.